Decred Wallet/Features/TransactionDetails/Cells/TransactionDetailCell.swift

```diff
@@ -1,38 +1,47 @@
 //  TransactionDetailCell.swift
 //  Decred Wallet
 //
-// Copyright (c) 2018-2019 The Decred developers
+// Copyright (c) 2018-2020 The Decred developers
 // Use of this source code is governed by an ISC
 // license that can be found in the LICENSE file.
 
 import UIKit
 
+struct TransactionDetails {
+    let title: String
+    let value: NSAttributedString
+    let isCopyEnabled: Bool
+}
+
 class TransactionDetailCell: UITableViewCell {
-    
-    @IBOutlet private weak var labelTitle: UILabel!
-    @IBOutlet private weak var labelValue: UILabel!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var valueButton: UIButton!
+    var presentingController: TransactionDetailsViewController?
     
     var txnDetails: TransactionDetails? {
         didSet {
             showData()
         }
     }
-    
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
-    }
-    
-    override func setSelected(_ selected: Bool, animated: Bool) {
-        super.setSelected(selected, animated: animated)
-        // Configure the view for the selected state
+
+    @IBAction func onValueButtonClicked(_ sender: Any) {
+        if let txn = self.txnDetails,
+            txn.isCopyEnabled,
+            let presentingController = self.presentingController {
+            DispatchQueue.main.async {
+                //Copy a string to the pasteboard.
+                UIPasteboard.general.string = txn.value.string
+                Utils.showBanner(parentVC: presentingController, type: .success, text: String(format: LocalizedStrings.sgCopied, txn.title))
```

Comment:
As a follow up of the previous comment, this becomes:
```suggestion
                self.onTxDetailValueCopied?(txDetail.title)
```